electrum/plugins/coldcard/qt.py

```diff
@@ -48,23 +52,59 @@ def wallet_info_buttons(self, main_window: 'ElectrumWindow', dialog):
         # user is about to see the "Wallet Information" dialog
         # - add a button if multisig wallet, and a Coldcard is a cosigner.
         assert isinstance(main_window, ElectrumWindow), f"{type(main_window)}"
+
+        buttons = []
         wallet = main_window.wallet
 
         if type(wallet) is not Multisig_Wallet:
             return
 
-        if not any(type(ks) == self.keystore_class for ks in wallet.get_keystores()):
+        coldcard_keystores = [
+            ks
+            for ks in wallet.get_keystores()
+            if type(ks) == self.keystore_class
+        ]
+        if not coldcard_keystores:
             # doesn't involve a Coldcard wallet, hide feature
             return
 
-        btn = QPushButton(_("Export for Coldcard"))
-        btn.clicked.connect(lambda unused: self.export_multisig_setup(main_window, wallet))
+        btn_export = QPushButton(_("Export multisig for Coldcard as file"))
+        btn_export.clicked.connect(lambda unused: self.export_multisig_setup(main_window, wallet))
+        buttons.append(btn_export)
+        btn_import_usb = QPushButton(_("Export multisig to Coldcard via USB"))
+        btn_import_usb.clicked.connect(lambda unused: self.import_multisig_wallet_to_cc(main_window, coldcard_keystores))
+        buttons.append(btn_import_usb)
+        return buttons
 
-        return btn
+    def import_multisig_wallet_to_cc(self, main_window, coldcard_keystores):
+        from io import StringIO
+        from ckcc.protocol import CCProtocolPacker
 
-    def export_multisig_setup(self, main_window, wallet):
+        index = main_window.query_choice(
+            _("Please select which {} device to use:".format(self.device)),
```

Comment:
```suggestion
            _("Please select which {} device to use:").format(self.device),
```
and same issue everywhere

Reply:
Ah yes. Interestingly this still seems to work (?)

Reply:
What do you mean by "work"? My claim is that `_("Please select which {} device to use:".format(self.device))` cannot be translated. I don't expect other runtime problems besides that.

I think these two are equivalent:
```
(1)  f"My name: {name}"
(2)  "My name: {name}".format(name=name)
```
Meaning these two are equivalent:
```
(3)  _(f"My name: {name}")
(4)  _("My name: {name}".format(name=name))
```

And regarding (3), we clearly have a comment saying that does not work, so (4) should not work either:
https://github.com/spesmilo/electrum/blob/ffdd45ee6f7d49501582e4c935c17acd1a5a75ac/electrum/i18n.py#L46-L48


Or do you mean you tested translations and even that works like that? If so, I don't understand how.

Reply:
Heh, no need for deep semantic discussions, 'seems to work' was merely meant as not crashing